assets/src/edit-story/elements/video/output.js

```diff
@@ -45,6 +45,7 @@ function VideoOutput({
     height: `${editorPixels((imgProps.height / height) * 100)}%`,
     left: `${-editorPixels((imgProps.offsetX / width) * 100)}%`,
     top: `${-editorPixels((imgProps.offsetY / height) * 100)}%`,
+    opacity: opacity / 100,
```

Comment:
Let's move this higher up to the `OutputElement` - it has a wrapper that should work for this.

Reply:
Which existing wrapper did you have in mind? There is currently a wrapper for mask and for the link.

Looks like as of today's updates, the Opacity layer is now applied to all the elements, it does make sense to have an extra opacity wrapper in this case, since (based on Figma) the color and background opacity needs to be handled for the element itself as well, e.g. for the Shape and Text.

Reply:
Yes. The mask is the right level. The mask is intentionally the top-level wrapper of the element. It handles clipping, and opacity is very related to the clipping.

Reply:
Btw, still seeing it here.